packages/angular/src/generators/component/schema.json

```diff
@@ -9,17 +9,16 @@
   "properties": {
     "path": {
       "type": "string",
-      "description": "The path at which to create the component file, relative to the workspace root. By default, it is set to the root of the project.",
+      "description": "The file path to the component without the file extension and suffix (e.g. `my-app/src/app/my-component/my-component` generates the file `my-app/src/app/my-component/my-component.component.ts`). Relative to the current working directory.",
       "$default": {
         "$source": "argv",
         "index": 0
       },
-      "x-prompt": "Where to create the component?"
+      "x-prompt": "What is the component file path?"
```

Comment:
`Where to generate the component?` ?

Reply:
I considered that one, but it is less explicit about the fact that the option is meant to be a file path. I find it more ambiguous. While the current one explicitly asks for a file path.